VideoWeb/VideoWeb/ClientApp/src/app/services/api/video-web.service.ts

```diff
@@ -36,6 +36,7 @@ export class VideoWebService {
     }
 
     getConferenceById(conferenceId: string): Observable<ConferenceResponse> {
+        console.trace('boop de bop');
```

Comment:
What is this Shaed :)

Reply:
haha i needed to figure out which test was calling the API rather thana stub

Reply:
removed :)